nilearn/datasets/utils.py

```diff
@@ -81,8 +84,10 @@ def _chunk_report_(bytes_so_far, total_size, initial_size, t0):
         If not resuming, set to zero.
     """
 
+    if cur_chunk_size == 0:
+        pass  # sys.stderr.write('\n')
```

Comment:
This newline was emitted elsewhere. It seems odd to have a reporting hook... and then not put as much of the reporting code as possible inside of it.